CMakeLists.txt

```diff
@@ -5,19 +5,30 @@ PROJECT(qmcpack)
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.10)
 IF(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
+  IF(NOT($ENV{CRAYPE_VERSION} MATCHES "."))
```

Comment:
This test is backwards - the 'NOT' should be removed.

Reply:
Message was backwards.  The policy needs to not be attempted on CRAY, at least with cmake 2.8.10 it causes a failure.